packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -246,6 +253,7 @@ export class CdkToolkit {
           });
         }
       }
+      print('\n✨  Total time: %ss', roundPercentage((elapsedSynthTime + elapsedDeployTime) / 1000));
```

Comment:
I notice that the `synth` time has a newline at the end, but neither deploy time nor total time do. Is that an intentional difference between them?

Reply:
added newline to both. I like the clarity of an extra new line, and originally `deploy` did not have it because the following print statements began with a `\n` whereas the ones after `synth` did not. However, I see this as an opportunity to remove those extraneous new lines.